lib/EmulationManager.js

```diff
@@ -33,13 +33,13 @@ class EmulationManager {
     const width = viewport.width;
     const height = viewport.height;
     const deviceScaleFactor = viewport.deviceScaleFactor || 1;
+    /** @type {Protocol.Emulation.ScreenOrientation} */
     const screenOrientation = viewport.isLandscape ? { angle: 90, type: 'landscapePrimary' } : { angle: 0, type: 'portraitPrimary' };
 
     await Promise.all([
       this._client.send('Emulation.setDeviceMetricsOverride', { mobile, width, height, deviceScaleFactor, screenOrientation }),
       this._client.send('Emulation.setTouchEmulationEnabled', {
-        enabled: viewport.hasTouch || false,
-        configuration: viewport.isMobile ? 'mobile' : 'desktop'
```

Comment:
why is `configuration` removed?

Reply:
There is no such option on `Emulation.setTouchEmulationEnabled`. It is only on `Page.setTouchEmulationEnabled`.